src/Microsoft.Fx.Portability/Analysis/AnalysisEngine.cs

```diff
@@ -221,6 +224,74 @@ public IEnumerable<string> FindUnreferencedAssemblies(IEnumerable<string> unrefe
             }
         }
 
+        public IList<NuGetPackageInfo> GetNuGetPackagesInfo(IEnumerable<string> assemblies, IEnumerable<FrameworkName> targets)
+        {
+            var nugetPackages = new List<NuGetPackageInfo>();
+            foreach (var assembly in assemblies)
+            {
+                ImmutableDictionary<FrameworkName, IEnumerable<NuGetPackageId>> packages = null;
+                var result = _packageFinder.FindPackage(assembly, targets, out packages);
+                if (result)
```

Comment:
No need for the `result` intermediate value

Reply:
Fixed.